tests/test_enrollment_api.py

```diff
@@ -55,3 +57,8 @@ def test_enroll_user_in_course():
     assert actual_response == expected_response
     request = responses.calls[0][0]  # pylint: disable=no-member
     assert json.loads(request.body) == expected_response
+
+
+def test_enroll_locally_raises():
+    with raises(NotConnectedToEdX):
```

Comment:
Yes, this is tested [here](https://github.com/edx/edx-enterprise/pull/25/files#diff-41cb95e2211fd63cab0a14339e33dc9eR107).